Libraries/Components/HyperlinkWindows/HyperlinkWindows.windows.js

```diff
@@ -0,0 +1,87 @@
+/**
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * All rights reserved.
+ *
+ * This source code is licensed under the MIT-style license found in the
+ * LICENSE file in the root directory of this source tree.
+ *
+ * @providesModule HyperlinkWindows
+ */
+'use strict';
+
+var PropTypes = require('prop-types');
+var React = require('React');
+var ReactNativeViewAttributes = require('ReactNativeViewAttributes');
+var ViewPropTypes = require('ViewPropTypes');
+var requireNativeComponent = require('requireNativeComponent');
+
+const ColorPropType = require('ColorPropType');
+const EdgeInsetsPropType = require('EdgeInsetsPropType');
+const StyleSheetPropType = require('StyleSheetPropType');
+const Text = require('Text');
+const TextStylePropTypes = require('TextStylePropTypes');
+const TouchableWithoutFeedback = require('TouchableWithoutFeedback');
+
+const stylePropType = StyleSheetPropType(TextStylePropTypes);
+
+class HyperlinkWindows extends React.Component {
```

Comment:
>HyperlinkWindows [](start = 6, length = 16)

Another option to consider is we could implement this as a `propType` inside `Text` for react-native-windows. Similar to the `isSecureTextEntry` prop for `TextInput`, we could have a `isHyperlink` prop and implement this entirely as logic of the `Text` component. 

I don't have any specific concerns with the approach of using a separate component though.